google/cloud/bigquery/magics/magics.py

```diff
@@ -512,7 +522,7 @@ def _cell_magic(line, query):
 
     Args:
         line (str): "%%bigquery" followed by arguments as required
-        query (str): SQL query to run
+        cell_body (str): SQL query or other command to run TODO
```

Comment:
Can you clarify here?

Reply:
Will update soon to resolve test failures - the TODOs are typos/oversights.